server/src/external/jellyfin/JellyfinApiClient.ts

```
@@ -359,6 +359,27 @@ export class JellyfinApiClient extends BaseApiClient {
     return `${this.options.url}/web/#/details?id=${id}`;
   }
 
+  async getGenres(
+    parentId?: string,
+    includeItemTypes?: string
+  ): Promise<QueryResult<string>> {
+    const genresResult = await this.doGet<string>({
+      url: `/Genres`,
+      params: {
+        parentId,
+        userId: this.options.userId,
+        includeItemTypes,
+        recursive: 'true',
+      },
+    });
+
+    if (genresResult instanceof Error) {
```

I just changed this recently to throw instead of return a Try - might be the cause of the build error

---

I resolved this on my end - will merge once it's fixed!